company/views.py

```diff
@@ -485,14 +496,23 @@ def get(self, request, id, *args, **kwargs):
     @validate_company_user
     @check_company_or_manager
     def post(self, request, id, *args, **kwargs):
+        domain_id = kwargs.get("domain_id")
         domain_data = {
             "name": request.POST.get("domain_name", None),
             "url": request.POST.get("domain_url", None),
             "github": request.POST.get("github_url", None),
             "twitter": request.POST.get("twitter_url", None),
             "facebook": request.POST.get("facebook_url", None),
+            "trademarkInput": request.POST.get("trademarkInput", None),
+            "remove-button": request.POST.get("removedTrademarks", None),
```

Comment:
The `remove-button` name looks odd.

Reply:
remove-trademark would be better for this I guess ?